src/api.v2/helpers/worker/workSubmit/submitMarkerHeatmapWork.js

```diff
@@ -1,23 +1,29 @@
+const bucketNames = require('../../../../config/bucketNames');
+const getS3Object = require('../../s3/getObject');
 const getExtraDependencies = require('./getExtraDependencies');
```

Comment:
a general question: why do we need helpers in the api like `submitMarkerHeatmapWork` or `submitEmbeddingWork`? Are these the hooks that you were talking about in this slack post: https://this-is-biomage.slack.com/archives/C015CHSUDRU/p1697808894319689 
There was time when the worker requests for the api were completely generic and we didn't need to distinguish between them. What happened?

Reply:
Yes, they are the hooks, related to this comment too https://github.com/hms-dbmi-cellenics/api/pull/446#discussion_r1367277163

Reply:
It is a shame, but I understand. It will be great to rework this (of course, outside of the scope of this PR).